packages/workbox-core/models/messages/messages.mjs

```diff
@@ -59,7 +59,7 @@ export default {
     }
     return `The parameter '${paramName}' passed into ` +
       `'${moduleName}.${className}.${funcName}()' must be an instance of ` +
-      `class ${expectedClass}.`;
+      `class ${expectedClass.name}.`;
```

Comment:
This was just something random I ran into when testing out the new code—you need the `.name` or else it doesn't log anything meaningful.